chaco/legend.py

```diff
@@ -487,29 +488,27 @@ def _composite_icon_renderer_default(self):
         return CompositeIconRenderer()
 
     # -- trait handlers --------------------------------------------------------
-    def _anytrait_changed(self, name, old, new):
-        if name in (
-            "font",
-            "border_padding",
-            "padding",
-            "line_spacing",
-            "icon_bounds",
-            "icon_spacing",
-            "labels",
-            "plots",
-            "plots_items",
-            "labels_items",
-            "border_width",
-            "align",
-            "position",
-            "position_items",
-            "bounds",
-            "bounds_items",
-            "label_at_top",
-        ):
-            self._layout_needed = True
-        if name == "color":
-            self.get_preferred_size()
+    @observe([
+        "font",
+        "border_padding",
+        "padding",
+        "line_spacing",
+        "icon_bounds",
+        "icon_spacing",
+        "labels.items",
+        "plots.items",
+        "border_width",
+        "align",
+        "position.items",
+        "bounds.items",
+        "title_at_top",
```

Comment:
```suggestion
        "label_at_top",
```

Reply:
I believe this had actually previously been a bug. There is no `label_at_top` trait defined on this class, but there is a `title_at_top`

Reply:
ahh. never mind then. please go ahead.